examples/3d/load_gltf.rs

```diff
@@ -21,18 +21,8 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
         transform: Transform::from_xyz(0.7, 0.7, 1.0).looking_at(Vec3::new(0.0, 0.3, 0.0), Vec3::Y),
         ..default()
     });
-    const HALF_SIZE: f32 = 1.0;
     commands.spawn(DirectionalLightBundle {
         directional_light: DirectionalLight {
```

Comment:
Probably also tweak the cascades here so this one looks good. It's kind of the fanciest-looking in-repo PBR example.